Modules/Microsoft365DSC/DSCResources/MSFT_AADRoleSetting/Readme.md

```diff
@@ -0,0 +1,18 @@
+# AADRoleSetting
+
+## Description
+
+This resource configure existing Azure roles. All UI parameters can be configured using this resource like:
+- Notifications
+- require approval / ticket / justification / MFA
+
+## Azure AD Permissions
```

Comment:
Please remove this Azure AD Permissions section. This is documented in the Settings.json and then used to automatically generate the documentation content.